llm_bench/python/utils/conversion_utils/helpers.py

```diff
@@ -2,14 +2,17 @@
 # Copyright (C) 2023-2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import logging as log
+import warnings
 from enum import Enum
 from pathlib import Path
-import logging as log
+from typing import Union
+
 import torch
 from nncf import compress_weights
 from openvino import save_model
+from optimum.intel.utils.import_utils import is_openvino_tokenizers_available
```

Comment:
please move optimum-intel commit in requirements.txt on one which contains ov tokenizers support